### ISSUE TITLE
[Question] Is the 1.21 version compatible with 1.21.1?

### ISSUE THREAD

**Issue by DarthCaesium** [0rc1nus/Galosphere-Main#120]
Hi! As the above title says, are the versions made for 1.21 compatible with 1.21.1? Also, will you be making versions for NeoForge going forward as well?

**Comment by SkyRunner55** [comment]
i'm also down for neoforge to come out

**Comment by SkyRunner55** [comment]
i'm curious when is gonna be updated to neoforge, most of big mods are migrating to neoforge (last exemple : L_Ender 's Cataclysm)

**Comment by Poopooracoocoo** [comment]
Yes. 1.21.1 was just a hotfix. All the mod dev needs to do is change the Modrinth and CurseForge metadata

**Comment by SkyRunner55** [comment]
> Yes. 1.21.1 was just a hotfix. All the mod dev needs to do is change the Modrinth and CurseForge metadata

still no neoforge version which is sad, all big mod like cataclysm & create are moving to neoforge so
